pkg/cmd/server/bootstrappolicy/securitycontextconstraints.go

```diff
@@ -285,15 +285,15 @@ func GetBootstrapSecurityContextConstraints(sccNameToAdditionalGroups map[string
 // GetBoostrapSCCAccess provides the default set of access that should be passed to GetBootstrapSecurityContextConstraints.
 func GetBoostrapSCCAccess(infraNamespace string) (map[string][]string, map[string][]string) {
 	groups := map[string][]string{
-		SecurityContextConstraintPrivileged: {ClusterAdminGroup, NodesGroup},
+		SecurityContextConstraintPrivileged: {ClusterAdminGroup, NodesGroup, MastersGroup},
```

Comment:
@deads2k do we want to go ahead an remove `ClusterAdminGroup`?

Reply:
> @deads2k do we want to go ahead an remove ClusterAdminGroup

Mark the constant deprecated, but I wouldn't remove it.  If people used it (and it looks reasonable to use), a tighten shouldn't break them.